.gitignore

```diff
@@ -46,10 +46,13 @@ exts/cesium.omniverse/cesium/omniverse/bindings/CesiumOmniversePythonBindings.cp
 exts/cesium.usd.plugins/cesium/usd/plugins/CesiumUsdSchemas/_CesiumUsdSchemas.so
 exts/cesium.usd.plugins/cesium/usd/plugins/CesiumUsdSchemas/_CesiumUsdSchemas.pyd
 exts/cesium.usd.plugins/cesium/usd/plugins/CesiumUsdSchemas/_CesiumUsdSchemas.lib
+exts/cesium.tests/cesium/tests/bindings/CesiumOmniverseTestsPythonBindings.cpython-310-x86_64-linux-gnu.so
+exts/cesium.tests/cesium/tests/bindings/CesiumOmniverseTestsPythonBindings.cp310-win_amd64.pyd
 
 # Installed libraries
 exts/cesium.omniverse/bin/
 exts/cesium.usd.plugins/bin/
+exts/cesium.tests/bin/
```

Comment:
I think the kit extension should be renamed to `cesium.omniverse.tests` so that it's clear that these are tests for the `cesium.omniverse` extension.

Reply:
Note: as part of this you will need to rename `cesium.omniverse.tests.kit` to something else otherwise the name conflict will cause issues. Maybe `cesium.omniverse.tests.runner.kit`.

Reply:
I think the omniverse component of the name is unnecessary and deviates from the convention used in our other extensions. These extensions only exist in a context where the relationship to omniverse is clear: in the cesium-omniverse repo or (if deployed) in Omniverse itself.

Reply:
I'm not sure if this is future proof though. We may want to publish other extensions and write tests for those, in which case it's not clear what `cesium.tests` is testing.

I could imagine eventually

`cesium.powertools` -> `cesium.powertools.tests`
`cesium.usd.plugins` -> `cesium.usd.plugins.tests`
`cesium.extensionInOtherRepo` -> `cesium.extensionInOtherRepo.tests`

There's already some precedence for the `.tests` suffix

![image](https://github.com/CesiumGS/cesium-omniverse/assets/915398/b305c135-f17f-4afd-9795-2c7a529679f7)

This change would also make the extension name and carbonite plugin name more consistent, as we call the carbonite plugin `cesium.omniverse.tests.plugin` and the library `CesiumOmniverseTests`.

Reply:
That said, it's pretty rare to see kit extensions have a separate tests extension except the few examples in the screenshot above above. So maybe we need to reevaluate this at some point.

Reply:
Done. I needed to change the name to `cesium.omniverse.cpp.tests` to avoid a conflict with the python module in `cesium.omniverse/tests`